### PR TITLE
Report: add legend to decipher iconography left to us by the ancient ones

### PR DESCRIPTION
**Pull request by ebidel**
R: all

Time for a legend kids! As we've added more icons, it's become unclear what everything means.

screen shot 2017-03-08 at 5 23 48 pm

### CODE REVIEW

File: lighthouse-core/report/styles/report.css [hunk 1]
```diff
@@ -120,7 +126,6 @@ summary {
   margin: 0 auto;
   max-width: var(--report-width);
   background-color: #fff;
-  box-shadow: 0 0 6px 0 rgba(0,0,0,0.26);
```
Comment (ebidel) [inline lighthouse-core/report/styles/report.css]: this is no longer visible, so removing.

**Comment by paulirish** [comment]
My my this is very attractive. Nice stuff, man.

File: lighthouse-core/report/templates/report-template.html [hunk 2]
```diff
@@ -226,7 +226,80 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
...
+                  <span class="aggregation__header__score score-good-bg subitem-result__good">Good</span>
+                  <span class="footer__key-description">Overall category score is 75-100</span>
+                </li>
+                <li>
+                  <span class="aggregation__header__score score-average-bg subitem-result__good">Average</span>
+                  <span class="footer__key-description">Overall category score is 45-74</span>
+                </li>
+                <li>
+                  <span class="aggregation__header__score score-poor-bg subitem-result__poor">Poor</span>
+                  <span class="footer__key-description">Overall category score is < 73</span>
```
Comment (sendilkumarn) [inline lighthouse-core/report/templates/report-template.html]: is it < 45 ?

Comment (ebidel) [inline lighthouse-core/report/templates/report-template.html]: Good catch. Done

**Comment by ebidel** [comment]
> I'd normally argue the fact that we need a legend for report scoring is a UX smell and an argument that we need to fix the report itself, but that's in progress so hopefully this isn't a permanent solution 

100% agree. It's gotten out of control. I wanted to get this in before the report redo. No telling how long or what form that would take.

**Pull request merged**